[#73529] 17.3 Group administration changes  https://community.openpro… (#22588)
opf/openproject: docs/system-admin-guide/users-permissions/groups/README.md

![create groups in OpenProject administration](openproject_system_guide_groups.png)

> Give your new group a unique name. Optionally, you can assign it to a parent group. This places the group within a hierarchy, where it automatically adopts the members and permissions of the parent group. The default setting is *No parent group*.

Reviewer: Members might be confusing here. The child groups also becomes a member of the project's that the parent is a member of. 

But the users that are group members in the parent are not automatically becoming members of the parent group. Maybe we can reword.

Author: How about this, @klaustopher 
.... Optionally, you can assign it to a parent group. This places the group within a hierarchy, where it inherits project memberships and permissions from the parent group. The default setting is No parent group.

> Give your new group a unique name. Optionally, you can assign it to a parent group. This places the group within a hierarchy, where it automatically adopts the members and permissions of the parent group. The default setting is *No parent group*.

Author: ```suggestion
Give your new group a unique name. Optionally, you can assign it to a parent group. This places the group within a hierarchy, where it inherits project memberships and permissions from the parent group. The default setting is *No parent group*.
```
